python/sparkdl/graph/spark_utils.py

```diff
@@ -0,0 +1,108 @@
+#
```

Comment:
I intend to consider things under graph/ as more like developer tools whereas things under udf/ are baked UDFs that end users can use.